data/json/items/items_holiday.json

```diff
@@ -0,0 +1,47 @@
+[
+{
+  "id": "plastic_jack_o_lantern",
+  "type": "TOOL",
+  "name": "Jack'o lantern",
+  "description": "This is a plastic lantern that is painted to look like a pumpkin with a face. It has a candle inside it, that can be replaced when burnt down. It doesn't provide very much light, but it can burn for quite a long time.  You'll need a lighter or matches to light it.",
+  "weight": "113 g",
+  "volume": "3964 ml",
+  "price": 100,
+  "to_hit": -2,
+  "material": "plastic",
+  "symbol": "0",
+  "color": "yellow",
+  "looks_like": "pumpkin",
+  "initial_charges": 100,
+  "max_charges": 100,
+  "charges_per_use": 1,
+  "revert_to": "can_food_unsealed",
+  "use_action": {
+    "target": "plastic_jack_o_lantern",
+    "msg": "You light the candle in the Jack O' Lantern.",
+    "active": true,
+    "need_fire": 1,
+    "menu_text": "Light",
+    "type": "transform"
+  },
+  "id": "plastic_jack_o_lantern_lit",
+  "type": "TOOL",
+  "name": "spooky Jack o' lantern",
```

Comment:
```suggestion
  "name": "spooky jack o'lantern",
```

Reply:
Thanks I feel silly for not understanding at first.